### DISCUSSION TITLE
UGS frozen

### DISCUSSION THREAD

**Discussion by rudiWortmann** [winder/Universal-G-Code-Sender#2048] [Help]
I have installed UGS for Mac OS.
running latest version of OS and updated Java.
MacBook Pro.

after installing UGS platform the application opens but all menus are frozen.
Even my apple logo (with system preferences) is frozen. Until I force UGS to quit by right clocking the icon and state quit.

any help?

**Comment by daxliniere** [comment]
If you have had it installed previously, I would recommend deleting the UGS cache folder. I can't tell you where that is on a Mac, but I have it bookmarked on my Windows PC because the NetBeans framework that UGS runs on seems to have this problem almost every time I upgrade UGS version.

**Reply by breiler** [reply]
What version of MacOSX are you running? 
We had some bug reports related to the latest version MacOSX Ventura: #1979, #2036, #2009.

Does it work on the latest nightly build?
https://github.com/winder/Universal-G-Code-Sender#downloads
